src/classes/OPP_StageToStateMapping_API.cls

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
Is this still an `API` class? Generally we apply the `API` suffix to classes that are externally visible, but that's not the case here. I think this is probably just fine at `OPP_StageToStateMapping`, though will this class ever support functionality beyond just "Stage to State Mapping"? Basically, I'm wondering if this is more related to Abacus utilities than just "Stage to State Mapping"

Could also be something like:
* `OPP_StageMappingUtil`
* `OPP_StageMapper`
* `OPP_AbacusUtil`

Reply:
So, @pedrodalcol  is still working this out this some of the NGO team, but my understanding is that some of the Stage to State functionality may be desirable for NPSP as well as Abacus.  For example, it may be desirable to expose a setting that would allow a customer to automatically write off unpaid payments when an Opportunity becomes "Finalized".  I can refactor it to Util, but do not think it should become an Abacus centric class.

Reply:
Thanks. Based on your details I agree this is not an abacus centric class.

I still think a different class name is needed though. Maybe `OPP_StageMapper` or `OPP_StageMappingUtil`; the former if we go with an instance class, the latter if it stays static - if you agree.

Reply:
I would like to keep it a UTIL Class.  I tend to think of instance classes when different instances of the class (no pun intended) have/need their own private variables.  This does not/and it is undesirable.  The only variables supporting this process are static.  If you feel super strongly I will go the other route, but it feels like a static utility class to me...

I do get that there may come a time where we might want to do decorate an Opportunity with additional functionality (passing it into a constructor), but it is really a different use case than this is built for (and a separate class could be built for that, but still use the util class for this piece).

Reply:
Good explanation. Let's go with `OPP_StageMappingUtil` as the class name then.